Client/mods/deathmatch/logic/CClientMarker.cpp

```diff
@@ -540,3 +545,36 @@ void CClientMarker::SetIgnoreAlphaLimits(bool ignore)
 {
     m_pMarker->SetIgnoreAlphaLimits(ignore);
 }
+
+bool CClientMarker::IsClientSideOnScreen() noexcept
```

Comment:
You should only use noexcept if the functions being called are also noexcept, so in this case these functions should not be marked as noexcept.